src/TypeScriptGenerator.ts

```diff
@@ -3,7 +3,7 @@ import {
   TypeGenerator,
   TypeGeneratorOptions
 } from "relay-compiler";
-import * as RelayCompilerPublic from "relay-compiler/lib/RelayCompilerPublic";
+import * as RelayCompilerPublic from "relay-compiler/lib";
```

Comment:
Can we use the main module instead of importing `/lib` (`import * as RelayCompilerPublic from "relay-compiler/lib"`)?